### PR TITLE
Images commands

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9370

## What
- Fix Outdated info in the README
- Replace _ for - in order to follow the common standard to make commands
- Add init release tag

## Verification Steps
- Check that the CI worked 
- Check the changes in the readme 

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,15 +7,15 @@ jobs:
...
-      - image: circleci/golang:1.10
-      - image: registry.access.redhat.com/rhscl/postgresql-96-rhel7:latest
-        ports:
-          - 5432:5432
-        environment:
-          POSTGRESQL_PASSWORD: postgres
-          POSTGRESQL_USER: postgresql
-          POSTGRESQL_DATABASE: mobile_security_service
-    
+    - image: circleci/golang:1.10
```
Comment (damienomurchu) [inline .circleci/config.yml]: I think we should update this - wdyt?

File: .circleci/config.yml [hunk 2]
```diff
@@ -55,9 +55,9 @@ jobs:
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make setup
       - setup_remote_docker
-      - run: make build_linux
-      - run: make build_master_image
-      - run: make push_master_image
+      - run: make build-linux
```
Comment (damienomurchu) [inline .circleci/config.yml]: Is there are reason for the change from `_` to `-` ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -69,37 +69,37 @@ jobs:
...
+        filters:
+          tags:
+            only: /.*/
+    - build_ui:
+        filters:
+          tags:
+            only: /.*/
+    - push_master_image:
+        requires:
+        - build_server
```
Comment (damienomurchu) [inline .circleci/config.yml]: Should this not be indented?

File: Makefile [hunk 4]
```diff
@@ -8,36 +8,36 @@ PACKAGES     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
                    -exec dirname {} \\; | sort | uniq")
 BIN_DIR := $(GOPATH)/bin				   
 BINARY ?= mobile-security-service
+RELEASE_TAG = 0.0.1
```
Comment (damienomurchu) [inline Makefile]: This needs to be `RELEASE_TAG ?= $(CIRLE_TAG)` so when its run by Circle that the release tag is the tag that is on the git commit. Process for making a release is to tag the git commit and push it, CircleCI will then pick that up and build and push the image.

File: Makefile [hunk 5]
```diff
@@ -8,36 +8,36 @@ PACKAGES     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
...
 
 # This follows the output format for goreleaser
 BINARY_LINUX_64 = ./dist/linux_amd64/$(BINARY)
 
 IMAGE_REGISTRY=quay.io
-DOCKER_LATEST_TAG = $(IMAGE_REGISTRY)/$(ORG_NAME)/$(APP_NAME):latest
-DOCKER_MASTER_TAG = $(IMAGE_REGISTRY)/$(ORG_NAME)/$(APP_NAME):master
-RELEASE_TAG ?= $(CIRLE_TAG)
-DOCKER_RELEASE_TAG = $(IMAGE_REGISTRY)/$(ORG_NAME)/$(APP_NAME):$(RELEASE_TAG)
+LATEST_TAG = $(IMAGE_REGISTRY)/$(ORG_NAME)/$(APP_NAME):latest
```
Comment (damienomurchu) [inline Makefile]:
```suggestion
IMAGE_LATEST_TAG = $(IMAGE_REGISTRY)/$(ORG_NAME)/$(APP_NAME):latest
```

File: Makefile [hunk 6]
```diff
@@ -8,36 +8,36 @@ PACKAGES     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
...
+MASTER_TAG = $(IMAGE_REGISTRY)/$(ORG_NAME)/$(APP_NAME):master
```
Comment (damienomurchu) [inline Makefile]:
```suggestion
IMAGE_MASTER_TAG = $(IMAGE_REGISTRY)/$(ORG_NAME)/$(APP_NAME):master
```

**Comment by damienomurchu** [comment]
@camilamacedo86 not sure of a few things, all detailed in comments.

File: README.adoc [hunk 7]
```diff
@@ -397,8 +397,8 @@ NOTE: The mock file generated by the dep contains comments which will help you t
 | `make vet`                    | Examines source code and reports suspicious constructs using https://golang.org/cmd/vet/[vet]
 | `make fmt`                    | Formats code using https://golang.org/cmd/gofmt/[gofmt]
 | `make clean`                  | Removes binary compiled using `make build`
-| `make docker_push_release`    | Pushes release image to Docker image hosting repository
-| `make docker_push_master`     | Pushes master image to Docker image hosting repository
+| `make docker-push-release`    | Pushes release image to Docker image hosting repository
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
| `make docker-push-release`    | Pushes release image to image hosting repository
```

File: README.adoc [hunk 8]
```diff
@@ -397,8 +397,8 @@ NOTE: The mock file generated by the dep contains comments which will help you t
...
+| `make docker-push-master`     | Pushes master image to Docker image hosting repository
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
| `make docker-push-master`     | Pushes master image to image hosting repository
```

File: Makefile [hunk 9]
```diff
@@ -48,32 +48,32 @@ setup_githooks:
...
-.PHONY: build_linux
-build_linux: setup
+.PHONY: build-linux
+build-linux: setup
 	env GOOS=linux GOARCH=amd64 go build -o $(BINARY_LINUX_64) $(APP_FILE)
 
-.PHONY: docker_build
-docker_build: build_linux
-	docker build -t $(DOCKER_LATEST_TAG) --build-arg BINARY=$(BINARY_LINUX_64) .
+.PHONY: build-image
```
Comment (damienomurchu) [inline Makefile]: @camilamacedo86 do we need this Make command?

File: .circleci/config.yml (hunk 2 above)
Comment (camilamacedo86) [reply .circleci/config.yml]: Usually, the commands are `make value-value` so just to keep the same standard.

File: Makefile (hunk 4 above)
Comment (camilamacedo86) [reply Makefile]: Done

File: README.adoc (hunk 7 above)
Comment (camilamacedo86) [reply README.adoc]: Done

File: README.adoc (hunk 8 above)
Comment (camilamacedo86) [reply README.adoc]: Done

File: Makefile (hunk 9 above)
Comment (camilamacedo86) [reply Makefile]: usually commands use - instead of .. just trying to follow the common standard.

File: Makefile (hunk 6 above)
Comment (camilamacedo86) [reply Makefile]: Done

File: Makefile (hunk 5 above)
Comment (camilamacedo86) [reply Makefile]: Done

File: Makefile [hunk 10]
```diff
@@ -8,36 +8,36 @@ PACKAGES     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
                    -exec dirname {} \\; | sort | uniq")
 BIN_DIR := $(GOPATH)/bin				   
 BINARY ?= mobile-security-service
+RELEASE_TAG = $(CIRCLE_TAG)
```
Comment (damienomurchu) [inline Makefile]: :+1:

File: .circleci/config.yml (hunk 3 above)
Comment (damienomurchu) [reply .circleci/config.yml]: Just checked the CircleCI docs - the indentation levels in the previous file [are what they suggest](https://circleci.com/docs/2.0/configuration-reference/#section=configuration), for that reason I would suggest we don't change this.

File: .circleci/config.yml [hunk 11]
```diff
@@ -10,12 +10,12 @@ jobs:
       - image: circleci/golang:1.10
```
Comment (damienomurchu) [inline .circleci/config.yml]: Should we update this image @camilamacedo86 - I believe `circleci/golang:1.12` is available for use

File: .circleci/config.yml (hunk 3 above)
Comment (camilamacedo86) [reply .circleci/config.yml]: it is equals before

File: .circleci/config.yml (hunk 11 above)
Comment (camilamacedo86) [reply .circleci/config.yml]: done

**Comment by camilamacedo86** [comment]
@damienomurchu I think all is done now, could you please re-check this one?

File: README.adoc [hunk 12]
```diff
@@ -397,8 +397,8 @@ NOTE: The mock file generated by the dep contains comments which will help you t
 | `make vet`                    | Examines source code and reports suspicious constructs using https://golang.org/cmd/vet/[vet]
 | `make fmt`                    | Formats code using https://golang.org/cmd/gofmt/[gofmt]
 | `make clean`                  | Removes binary compiled using `make build`
-| `make docker_push_release`    | Pushes release image to Docker image hosting repository
-| `make docker_push_master`     | Pushes master image to Docker image hosting repository
+| `make push-release-image`     | Pushes release image to  image hosting repository
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
| `make push-release-image`     | Pushes release image to image hosting repository

```

File: README.adoc [hunk 13]
```diff
@@ -397,8 +397,8 @@ NOTE: The mock file generated by the dep contains comments which will help you t
...
+| `make push-master-image`      | Pushes master image to Docker image hosting repository
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
| `make push-master-image`      | Pushes master image to image hosting repository
```

Comment (camilamacedo86) [reply README.adoc]: Done 👍

File: README.adoc (hunk 12 above)
Comment (camilamacedo86) [reply README.adoc]: Done

**Pull request merged**